docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -230,6 +230,8 @@ not meaningful for the `stop` subcommand.
   `${PWD}/kubestellar-logs`.
 - `--local-kcp $bool`: says whether to expect to find a local process
   named "kcp".  Defaults to "true".
+- `--ensure-imw`: provide a comma separated list of fully qualified inventory workspaces, _e.g._ "root:imw1,root:test:imw2". Defaults to "root:imw1". To prevet the creation of any inventory workspace, then pass "".
```

Comment:
(A) Elsewhere the terminology of pathname is used, rather than fully qualified.

(B) Since we have removed the use of hierarchy, let's give only flat examples: "root:imw1,root:imw2".

Reply:
Ok, I think I made all the changes

Reply:
I think so too.